src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <React.Fragment>
+      <section className="Person">
+        <h2 className="Person__name">My name is {name}</h2>
+        {age === undefined ? null : <p className="Person__age">I am {age}</p>}
+        {isMarried ? (
+          <p className="Person__partner">
+            {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+          </p>
+        ) : (
+          <p className="Person__partner">I am not married</p>
+        )}
```

Comment:
Let's do the check inside a p tag so that will have it only once and I would move it to a variable for better readability
```
const someVar = check here
<p>{someVar}</p>
```